maxwell-partitions.properties.example

```diff
@@ -0,0 +1,4 @@
+#database name to partition mapping
+db1=1
```

Comment:
if we're going down this road, I'd suggest that we go ahead with a table mapping too.  I'd suggest:

```
# table `t1` in any database maps to partition 43
*.t1 = 43  
# all tables in `d1` database map to 34
d1.* = 34
# just d2.t2 
d2.t2 = 99
```